RESideMenu/RESideMenuController.m

```diff
@@ -395,6 +395,8 @@ - (void)buildLayersForAnimation
     transform.m34 = -1.0 / (contentView.bounds.size.height * 4.6666667);
 
     self.perspectiveAnimationLayer = animationLayer;
+    self.perspectiveAnimationLayer.shouldRasterize = YES;
+    self.perspectiveAnimationLayer.rasterizationScale = [UIScreen mainScreen].scale;
```

Comment:
Nice. How does this affect performance when displaying the map in the content view? In theory, it will rasterise only once, unless the contents change. It would be good to know we have validated that this is the case in the main app.